splunk-otel-android/src/main/java/com/splunk/rum/AppStartupTimer.java

```diff
@@ -44,15 +57,76 @@ Span start(Tracer tracer) {
         return appStart;
     }
 
+    /** Called when Activity is created. */
+    void startUiInit() {
+        if (uiInitStarted || isStartedFromBackground) {
+            return;
+        }
+        uiInitStarted = true;
```

Comment:
This isn't threadsafe...do we care? Can two activities get created at the same time and step on each other?

Reply:
This should be called from the UI thread, I wouldn't expect this to be called from multiple threads.